support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -288,9 +288,8 @@ function WeeklyCheckoutForm(props: PropTypes) {
 								hideLabel={true}
 							>
 								{days
-									// Don't render input if Christmas day or Christmas eve
-									.filter((day) => !formatMachineDate(day).endsWith('-12-25'))
-									.filter((day) => !formatMachineDate(day).endsWith('-12-24'))
+									// Don't show 2022-12-30 as this is not a valid publication date in 2022
+									.filter((day) => !formatMachineDate(day).endsWith('-12-30'))
```

Comment:
```suggestion
									.filter((day) => {
									    const invalidPublicationDates = ['-12-24', '-12-25', '-12-30'];
									    const date = formatMachineDate(day);
									    return !invalidPublicationDates.some(dateSuffix => date.endsWith(dateSuffix))
									})
```
I don't think it's a good idea to remove the block on Christmas Eve and Christmas day, but there's also no need to us to do multiple filters.